eventlog/service.go

```diff
@@ -20,11 +20,16 @@ const contractName = "eventlog"
 const logCmd = "log"
 
 // Set a relatively low time for bucketMaxAge: during peak message arrival
-// this will pretect the buckets from getting too big. During low message
+// this will protect the buckets from getting too big. During low message
 // arrival (< 1 per 5 sec) it does not create extra buckets, because time
 // periods with no events do not need buckets created for them.
 const bucketMaxAge = 5 * time.Second
 
+// eventDelayTolerance indicates how much time after the event timestamp
+// we are stilling willing to accept an event as genuine and contemporaneous,
+// and hence as valid. Expressed in [ns]
+const eventDelayTolerance = time.Duration(-60 * 1e9)
```

Comment:
```suggestion
const eventDelayTolerance = time.Minute
```

And reverse the comment to say _before_, as well as changing the code.